test/org/sagebionetworks/bridge/models/schedules/CronTaskSchedulerTest.java

```diff
@@ -0,0 +1,120 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import static org.junit.Assert.assertEquals;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleTestUtils.dt;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleTestUtils.assertDates;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleType.ONCE;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleType.RECURRING;
+
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.Maps;
+
+/**
+ * Tests for the cron scheduler. Most details of the schedule object have been tested in 
+ * the IntervalTaskSchedulerTask class, here we're testing some of the specifics of 
+ * the cron schedules.  
+ */
+public class CronTaskSchedulerTest {
+    
+    private Map<String, DateTime> events;
+    private List<Task> tasks;
+    
+    private DateTime ENROLLMENT = DateTime.parse("2015-03-23T10:00:00Z");
```

Comment:
Can we change all our tests to run in a non-UTC time zone (like -0700) just to make sure we aren't accidentally forcing UTC anywhere?